docs/platform/study_conditions.json

```diff
@@ -52,6 +52,12 @@
                   "type": "integer",
                   "minimum": 0
                 },
+                "instrument_score": {
```

Comment:
we should maybe think about separating/branching out the resource_function stuff like we do for actions...lmk what u think